lms/templates/student_account/register.underscore

```diff
@@ -51,5 +51,7 @@
 
     <%= context.fields %>
 
-    <button type="submit" class="action action-primary action-update js-register register-button"><%- gettext("Create account") %></button>
+    <button type="submit" class="action action-primary action-update js-register register-button">
+    	<% if ( context.registerFormSubmitButtonText ) { %><%- context.registerFormSubmitButtonText %><% } else { %><%- gettext("Create account") %><% } %>
```

Comment:
Although this falling to the default case is unlikely, make sure account is capitalized, ie `Create Account`